drivers/sensor/aht20/KConfig

```diff
@@ -0,0 +1,20 @@
+# AHT20 temperature and humidity sensor driver configuration
```

Comment:
File name should be Kconfig (pretty sure CI will catch it anyway)

Reply:
Yes thanks, I just fixed